reactive/kotlinx-coroutines-reactor/src/FlowAsFlux.kt

```diff
@@ -0,0 +1,21 @@
+package kotlinx.coroutines.reactor
+
+import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flowOn
+import kotlinx.coroutines.reactive.flow.FlowSubscription
+import reactor.core.CoreSubscriber
+import reactor.core.publisher.Flux
+
+/**
+ * Transforms the given flow to [Flux].
+ */
+@ExperimentalCoroutinesApi
+public fun <T: Any> Flow<T>.asFlux(): Flux<T> = FlowAsFlux(this)
```

Comment:
Please provide more detailed documentation (like in `Publisher.asFlow`)

Reply:
Copied docs from Flow.asPublisher/asFlowable